src/bindgen/ir/cfg.rs

```diff
@@ -382,21 +382,26 @@ impl CfgWrite for Option<Cfg> {
                 return;
             }
 
-            out.write("#if ");
-            cfg.write_condition(config, out);
-            out.new_line();
+            let mut unmatched = vec![];
```

Comment:
I think you can use `&mut vec![]` below to avoid naming this if you want, or `matched_defined` could take an `Option`.